tools/VscPreviewExtension/src/constVariables/command.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+export const Command = {
+    previewCommand: "preview",
```

Comment:
They both use preview command.

---
In reply to: [111710657](https://github.com/dotnet/docfx/pull/1547#discussion_r111710657) [](ancestors = 111710657)